src/keys/utils.ts

```diff
@@ -41,18 +47,71 @@ async function generateKeyPair(bits: number): Promise<KeyPair> {
   return await generateKeyPair({ bits });
 }
 
+function generateMnemonic(words: 12 | 24 = 24): string {
```

Comment:
Ok so this generates the recovery code.

The mnemonic is the recovery code.

I think this should be called `generateRecoveryCode` to be clear.

This is a random operation. I checked that there are no asynchronous versions of that.

However the `generateMnemonic` has a rng option.

This `rng` option should be supplied:

```
(size: number) => Buffer
```

This has to be synchronous and by default it's using `crypto.randomBytes`, so we should be giving this our own random byte generator.

Reply:
Furthermore we should be explicit about what word list we are using.

So at the beginning of the `bip39` import. We should do this statically:

```ts
bip39.setDefaultWordlist('english');
```

Reply:
Example:

```ts
  const code = bip39.generateMnemonic(
    256,
    keysUtils.getRandomBytesSync
  );
```